File: OOP/Point.java
import java.util.Scanner;

public class Point {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int test = in.nextInt();
        while(test > 0){
            test--;
            double a = in.nextDouble();
            double b = in.nextDouble();
            double c = in.nextDouble();
            double d = in.nextDouble();
            coordinates p1 = new coordinates(a,b);
            coordinates p2 = new coordinates(c,d);
            System.out.format("%.4f\n", coordinates.distance(p1, p2));
        }
        in.close();
    }   
    static class coordinates{
        double x;
        double y;
        public coordinates(){

        }
        public coordinates(double x, double y){
            this.x = x;
            this.y = y;
        }
        public double distance(coordinates p){
            return Math.sqrt(Math.pow(this.x - p.x,2) + Math.pow(this.y - p.y,2));
        }
        public static double distance(coordinates p1,coordinates p2){
            return Math.sqrt(Math.pow(p1.x - p2.x,2) + Math.pow(p1.y - p2.y,2));
        }
    }
}

File: PythonPTIT/Point.py
import math


class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def distance(self, Point):
        d = math.sqrt((self.x - Point.x) ** 2 + (self.y - Point.y) ** 2)
        return "%.4f" % d


if __name__ == '__main__':
    n = int(input())
    while n > 0:
        x, y, z, t = map(int, input().split())
        a = Point(x, y)
        b = Point(z, t)
        print(a.distance(b))
        n -= 1
